### PR TITLE
Swiper: Implement typings for v4.2 of http://idangero.us/swiper

### PR DESCRIPTION
**Pull request by ematseruk**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: http://idangero.us/swiper/api/
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/swiper/index.d.ts [hunk 1]
```diff
@@ -1,12 +1,195 @@
-// Type definitions for Swiper 3.4
+// Type definitions for Swiper 4.2
 // Project: https://github.com/nolimits4web/Swiper
-// Definitions by: Sebastián Galiano <https://github.com/sgaliano>, Luca Trazzi <https://github.com/lucax88x>
+// Definitions by: Eugene Matseruk <https://github.com/ematseruk>
```
Comment (mhegazy) [inline types/swiper/index.d.ts]: please do not change the owner of the package. add your name to the list.

**Comment by mhegazy** [comment]
@sgaliano can you please review this change.

**Review by mhegazy — DISMISSED** [review]

**Comment by ematseruk** [comment]
@mhegazy, it looks like @sgaliano is not an active user. Can we ask another contributor @lucax88x for a review?

**Comment by lucax88x** [comment]
As soon I am on a pc I surely can

**Comment by lucax88x** [comment]
@mhegazy  For me it's ok, but I cannot approve myself as I'm not a reviewer

**Comment by RyanCavanaugh** [comment]
🌟 🎈 🎉 🏆 🎂 ✨ ⭐️ 

Congratulations on your first DefinitelyTyped contribution!

🌟 🎈 🎉 🏆 🎂 ✨ ⭐️

**Pull request merged**